app/controllers/orders_controller.rb

```diff
@@ -20,4 +21,13 @@ def create
 			redirect_to cart_path
 		end
 	end
+
+	def update
+		order = Order.find(params[:id])
+		order.cancelled! if params[:status] == 'cancelled'
```

Comment:
Yeah, we should've thought ahead to make this RESTful. Looks like you got it up and running though! Nice work